src/components/node.js

```diff
@@ -220,7 +223,7 @@ function Node({
         newState.x += off.xOff;
         newState.y += off.yOff;
         // now tell the graph that we're actually drawing an edge
-      } else if (layoutEngine) {
+      } else if (!draggingEdge.current && layoutEngine) {
```

Comment:
prior to #283 there were draggingEdge state references here, which worked with the class component (because the event handlers were static and referred to state via `this`)